### ISSUE TITLE
Fatal: Not supported ONNX opcode: BatchNormalization

### ISSUE THREAD

**Issue by 18310837090** [kendryte/nncase#172]
./ncc compile /home/cool/best.onnx /home/cool/best.kmodel -i onnx -o kmodel -t k210 --dataset demo.png
error:Fatal: Not supported ONNX opcode: BatchNormalization
can you tell me what problems it cause?
onnx not supposed BatchNormalization?

**Comment by alexcruz0202** [comment]
Could you solve the problem?
Recently, I have worked with Kendryte K210 and test my ONNX model. I have met same problem.
I have tried to fuse bn to conv in ONNX model. 
So I have used the ONNX simplifier and optimizer, but failed to do it.
Appreciate if someone can help me.

**Comment by PureHing** [comment]
@alexcruz0202 training your task by pytorch1.2 ,and convert to onnx, then execute onnxsim() function as following
```sh
def  onnxsim():
    import onnx
    from onnx import optimizer

    ori_model = onnx.load("src-model.onnx") 
    passes = ['fuse_bn_into_conv'] 
    optim_model = optimizer.optimize(ori_model, passes)
    onnx.save(optim_model, "mbc-sim.onnx")
    os.system("python -m onnxsim mbc-sim.onnx  mbc_s.onnx")
```

**Comment by alexcruz0202** [comment]
@PureHing Thank you for your help.

**Comment by alexcruz0202** [comment]
One question. Is it possible when I use pytorch 1.4?

**Comment by 18310837090** [comment]
> One question. Is it possible when I use pytorch 1.4?

The official source code actually supports bt op. I generate new tools by recompiling the source code. The latest version of current release is not the latest source code compilation version.

**Comment by alexcruz0202** [comment]
@18310837090 
On Windows 10, I have rebuild tools according to your suggestion.
Thank you.

**Comment by zhangyang2057** [comment]
BatchNormalization op for onnx has been supported in features/upgrade branch. 
If you met any problem, please feel free to summit an issue.
